libs/barista-components/stacked-series-chart/README.md

```diff
@@ -134,7 +135,7 @@ containing useful information that can be used inside the overlay's template
 | --------------- | -------------------------- | ---------------------------------------------------------------- |
 | `origin`        | `DtStackedSeriesChartNode` | Node passed by user in `series` array.                           |
 | `valueRelative` | `number`                   | Numeric percentage value based on this node vs sum of top level. |
-| `color`         | `DtColors | string`        | Color for this node in this state.                               |
+| `color`         | `DtColors                  | string`                                                          | Color for this node in this state. |
```

Comment:
Something weird happened with the formatting. Have you checked it's correct in the docs?